prompt.go

```diff
@@ -181,8 +186,8 @@ func (p *Prompt) Input() string {
 			p.renderer.BreakLine(p.buf, p.lexer)
 			p.tearDown()
 			os.Exit(code)
-		default:
-			time.Sleep(10 * time.Millisecond)
```

Comment:
You removed the default sleeping mechanism? I expect this to increase the CPU usage a lot?

Reply:
Go will block when channels have no data available, there won't be any CPU usage increase. Ironically, is the `default` case that cause the loop to go over an over (because there is always a case to be evaluated) and you have to add a `sleep` but without that, you don't need it

Reply:
No, I think it could actually improve it. 
If there is no default case, the for select will just block if none of the channels have any data.

Reply:
Nice!